frontend/src/layout/ErrorNetwork.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react'
+import { HedgehogOverlay } from 'lib/components/HedgehogOverlay/HedgehogOverlay'
+import { Button } from 'antd'
+import { ReloadOutlined } from '@ant-design/icons'
+
+export function ErrorNetwork(): JSX.Element {
+    return (
+        <div>
+            <h2>Network Error</h2>
+            <p>There was an issue loading the requested resource.</p>
+            <p>
+                <Button onClick={() => window.location.reload()}>
+                    <ReloadOutlined /> Reload the page!
+                </Button>
+            </p>
+            <HedgehogOverlay type="sad" />
```

Comment:
not sure we should add the old logo back in. I think the page would look OK without a logo

Reply:
Yeah, it looks cleaner without it. The 404 page will still be hogged